contracts/Vault.vy

```diff
@@ -798,7 +816,10 @@ def _issueSharesForAmount(to: address, amount: uint256) -> uint256:
         # Mint amount of shares based on what the Vault is managing overall
         # NOTE: if sqrt(token.totalSupply()) > 1e39, this could potentially revert
         precisionFactor: uint256 = self.precisionFactor
-        shares = precisionFactor * amount * totalSupply / self._totalAssets() / precisionFactor
+
+        lockedFundsRatio: uint256 = (block.timestamp - self.lastReport) * self.lockedProfitDegration
```

Comment:
Is this unused code?

Reply:
yes. fixed 9701bc74ff4199de723ff4c4e5e69c71e6ce40e1